server/plugins/renderer/responses/raw-response.spec.js

```diff
@@ -19,11 +19,12 @@ lab.describe('RawResponse', () => {
     const statusCode = 200;
     let request;
     let response;
-    let reply;
+    let h;
```

Comment:
what does `h` stand for ?

Reply:
@junedkazi From the docs "h" stand for HAPI.
They replaced the reply function with this "Response Toolkit" (h) in v17.